network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1196,13 +1195,15 @@ def exec(self):
                                       'No traffic is specified as forbidden.')
         return IntersectsQuery(self.config1, self.config2).exec(True)
 
-    def compute_query_output(self, query_answer, _):
-        if query_answer.bool_result:
-            query_output = f'{self.config2.name} does not forbid connections specified in {self.config1.name}:' \
-                           f'{query_answer.output_explanation}'
+    def compute_query_output(self, query_answer, cmd_line_flag=False):
+        res = not query_answer.numerical_result if cmd_line_flag else query_answer.bool_result
+        query_output = query_answer.output_result + '\n'
+        if query_answer.numerical_result == 0:
+            query_output += f'{self.config2.name} does not forbid connections specified in {self.config1.name}: ' \
```

Comment:
Output issue in forbids: when output is "The two configurations have different network topologies and thus are not comparable" or "There are no NetworkPolicies in the given forbids config. No traffic is specified as forbidden" ,
there is still additional appended output of "config2 does not forbid connections specified in config1" - though it is not relevant..

Reply:
Now there is still additional appended output of "config2 forbids connections specified in config1"